### ISSUE TITLE
Occupation information should be editable

### ISSUE THREAD

**Issue by aalih** [SORMAS-Foundation/SORMAS-Project#2289]
### Situation Description
SORMAS include a list of occupation activities that can't be edited as it is currently done with region or districts. The only way to "change" this list is to provide other labels in the translation files.

### Feature Description
Allow defining custom occupation activities as it is currently done with regions and districts

**Comment by MateStrysewskeSym** [comment]
@aalih What exactly do you mean by occupation activities?

**Comment by MateStrysewskeSym** [comment]
@aalih Yeah, I've realised that's what you meant, which is why I deleted my comment. :)

**Comment by OrgaLandkreisTuebingen** [comment]
A first step to solve the problem would be to show a specifying field for every predefined occupation and not just for a few.

**Comment by kwa20** [comment]
related to #5015
